jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/OdeTimJsonTopology.java

```diff
@@ -64,19 +72,42 @@ public boolean isRunning() {
    */
   public Topology buildTopology(String topic) {
     StreamsBuilder builder = new StreamsBuilder();
-    builder.table(topic,
-        Materialized.<String, String>as(Stores.inMemoryKeyValueStore("timjson-store")));
+
+    // Create a windowed store with a retention period of 1 hour
+    KStream<String, String> timStream = builder.stream(topic);
+
+    timStream.groupByKey()
+        .windowedBy(TimeWindows.ofSizeWithNoGrace(Duration.ofHours(1))) // 1 hour window
+        .reduce(
+            (aggValue, newValue) -> newValue, // only keep latest value
+            Materialized.<String, String, WindowStore<Bytes, byte[]>>as("timjson-windowed-store")
```

Comment:
suggestion (non-blocking): a good refactor here would be to pull this magic string up to a variable